R/lockbox.R

```diff
@@ -49,8 +49,10 @@ lockbox.list <- function(lock, env) {
 
   ## Find the packages whose version does not match the current library.
   mismatches <- vapply(lock, version_mismatch, logical(1))
+  autoinstall_packages <- vapply(lock, function(lock) is.autoinstall_package(lock), logical(1))
+  reload_these_packages <- unlist(Map(any, mismatches, autoinstall_packages))
```

Comment:
I would call this `load_these_packages`, since we are loading them for the first time.